tests/integration/suite/scheduler/metrics/basic.go

```diff
@@ -0,0 +1,187 @@
+/*
+Copyright 2024 The Dapr Authors
```

Comment:
Update with suggestions from other file.

Reply:
I think this is done